### ISSUE TITLE
Looking for co-maintainers

### ISSUE THREAD

**Issue by siordache** [beryx/badass-jlink-plugin#223]
I'm looking for co-maintainers who can help keep the plugin alive. If you're interested, please leave a comment here. Anyone who is willing to help is welcome!

**Comment by airsquared** [comment]
@siordache I'd be happy to help out.

**Comment by siordache** [comment]
That's great, thanks! I sent you an invitation.

**Comment by airsquared** [comment]
Thanks! Would it be ok if I bumped the minimum Java to Java 17? That's the first LTS where jpackage is not preview/incubator.

**Comment by siordache** [comment]
Definitely. Thanks for your help, it's really great to have you as a co-maintainer!

**Comment by airsquared** [comment]
Great! How would I go about releasing a new version of the plugin?

**Comment by siordache** [comment]
I configured the `GRADLE_PUBLISH_KEY` and `GRADLE_PUBLISH_SECRET` secrets and added a manual GitHub action for publishing the plugin to the Gradle Plugins Portal. I didn't test it though, so you may need to adjust it a bit.
Set `pluginReleaseBuild` to `true` in `gradle.properties`. You can play with it before publishing the release by setting the `pluginVersionLabel` to some value (for example `alpha-1`).

**Comment by airsquared** [comment]
Thanks! Released the next version.

**Comment by DaiYuANg** [comment]
I'd be happy to help out too

**Comment by xzel23** [comment]
Hi @siordache, @airsquared ,

there are several bugs that have been fixed since the last release and I think a new release would be a good thing.  If both of you do not have the time, I'd volunteer to become a co-maintainer and spin a new release.

List of fixed bugs I'd recommend to include in a new version:
- #244: secondary Launcher fails with application main class missing (Kotlin DSL) - PR is already merged
- #266: prepare merged jar dir task configure error in multi module project 
- #275: release Java 23 compatible version

**Comment by siordache** [comment]
Hi @xzel23 , I've sent you an invitation to become a maintainer. I'm very grateful for your help!
